### ISSUE TITLE
jq.ui.loadContentQueue and doingTransition issue.

### ISSUE THREAD

**Issue by dustmoo** [01org/appframework#228]
I am having an odd issue with my panels.

I am still trying to figure out how or why, but when I am loading a panel for the third or more times and try navigating back from the panel. The app would transition back only to immediately transition to the panel I just left.

So far in my investigations I have found that when I load a panel for the third time doingTransition is set to true so it adds the loading to the queue. Checking the queue shows the loadContent object is queued, and when I navigate back, it loads the content and then transitions back because of the item in the queue, This clears the item from the queue and the second time I click back the transition is successful.

What is really interesting is that if I load that panel a fourth or subsequent times, it detects doingTransition multiple times before loading the content, each time making the queue larger  taking more "back" cycles to get out of the panel.

It seems that somewhere in the code doingTransition is being left as true.

This has only happens to my fourth panel deep into the app.

I am going to keep debugging, but wondering if you have run into this? 

TL;DR; Panels are being queued when they aren't supposed too leading to some sort of back loop where you can't get out of a panel without pressing back multiple times. The culprit seems to be "doingTransition".
